example/llm-finetune/models/chatglm3/finetune.py

```diff
@@ -72,6 +72,10 @@ class TrainArguments(Seq2SeqTrainingArguments):
     remove_unused_columns: bool = dataclasses.field(
         default=False, metadata={"help": "remove unused columns"}
     )
+    # Currently, we don't support `Optional[Union[str, Path, GenerationConfig]]` for the command line.
+    generation_config: t.Optional[str] = dataclasses.field(
```

Comment:
Does this property is unused?

Reply:
we need to change the default value for the command line. The HF trainer will use the property.

Reply:
ok，i see